### ISSUE TITLE
Flutter Material Icon - Question_mark 

### ISSUE THREAD

**Issue by abdullah017** [flutter/flutter#100137]
The question mark icon, which is the material icon, cannot be displayed on the application or appears as a different icon.

```dart
Column(
              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
              crossAxisAlignment: CrossAxisAlignment.center,
              children: [
                Row(children: const <Widget>[
                  Expanded(
                    flex: 1,
                    child: Icon(
                      Icons.question_mark_rounded,
                      color: Colors.blueAccent,
                    ),
                  ),
                  Expanded(
                    flex: 5,
                    // name
                    child: Text(
                        "Lorem ipsum dolor sit amet, consectetur adipiscing elit."),
                  ), // name
                ]),
              ],
            );
```

**Comment by danagbemava-nc** [comment]
Hi @abdullah017, please provide the output of `flutter doctor -v`. 

Thank you

**Comment by abdullah017** [comment]
`[✓] Flutter (Channel stable, 2.10.1, on macOS 12.3 21E230 darwin-arm, locale
    en-TR)
    • Flutter version 2.10.1 at /Users/piton/Documents/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision db747aa133 (5 weeks ago), 2022-02-09 13:57:35 -0600
    • Engine revision ab46186b24
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version
    32.1.0-rc1)
    • Android SDK at /Users/piton/Library/Android/sdk
    • Platform android-32, build-tools 32.1.0-rc1
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)

[✓] VS Code (version 1.65.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.36.0

[✓] Connected device (2 available)
    • sdk gphone64 arm64 (mobile) • emulator-5554 • android-arm64  • Android 12
      (API 32) (emulator)
    • Chrome (web)                • chrome        • web-javascript • Google
      Chrome 99.0.4844.74

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!`

**Comment by danagbemava-nc** [comment]
Hi @abdullah017, this has been fixed in flutter 2.10.3. Kindly upgrade to the latest version of flutter and let me know whether or not you still experience this issue. 

Thank you
